boards/nxp/rd_rw612_bga/Kconfig.defconfig

```diff
@@ -6,7 +6,7 @@
 if BOARD_RD_RW612_BGA
 
 config FLASH_LOAD_SIZE
-	default 0x400000 if !BOOTLOADER_MCUBOOT && !NXP_MONOLITHIC_BT
+	default 0x400000 if !BOOTLOADER_MCUBOOT && !(NXP_MONOLITHIC_WIFI || NXP_MONOLITHIC_BT)
```

Comment:
Not this way!
The Flash load size is defined by partitions.

Reply:
just refer to the change of NXP_MONOLITHIC_BT, you mean the original change is wrong?

Reply:
> you mean the original change is wrong?

This is a wrong approach. 
The issue was reported.

Reply:
Hi @butok 
I remove the commit you comment, please review again

Reply:
> Hi @butok I remove the commit you comment, please review again

Thank you, it should solve the the wrong partition size.